docs/source/kiva/quickref.rst

```diff
@@ -136,206 +283,3 @@ path_cmd:
 path_flags:
     path_flags, path_flags_ccw, path_flags_close, path_flags_cw,
     path_flags_mask, path_flags_none
-
-
-Graphics Context
-----------------
-
-Construction
-~~~~~~~~~~~~
-__init__(ary_or_size, pix_format="bgra32", interpolation="nearest", base_pixel_scale=1.0):
-    ``ary_or_size`` can be either a numpy array or a tuple of the form
-    (width, height). If it is an array, it will be used as the backing store
-    for the pixels. **Its shape must be compatible with ``pix_format``**
-
-    ``pix_format`` determines the pixel format and is a string which can be any
-    of the following: "gray8", "rgb555", "rgb565", "rgb24", "bgr24", "rgba32",
-    "argb32", "abgr32", "bgra32".
-
-    ``interpolation`` determines the interpolation used by scaled image drawing
```

Comment:
~not sure why we removed this from the docstring.~ looks like this isnt available consistently across the various supported backends.